donate.html

```diff
@@ -558,6 +592,24 @@ <h5 class="margin-bottom-normal">{% filter ak_text:"donate_billing_information"
     var arg_amount_selected = false;
     var arg_amount = {{ args.amount|default:"false" }};
 
+    // Records Product "Details View" based on product click from Homepage

```

Comment:
Event documentation for reference:
https://developers.google.com/analytics/devguides/collection/ua/gtm/enhanced-ecommerce#details